app/helpers/icons.py

```diff
@@ -0,0 +1,57 @@
+import os
+from flask import request
+
+from app.settings import IMAGE_FOLDER
+from app.settings import ROUTE_PREFIX
+
+
+def get_all_icons(red=255, green=0, blue=0):
```

Comment:
So all icons will have red as a default color, also the ones, that cannot be colorized, right?
Is that correct or do the "non-colorizable" icons each need to have their individual colors?

Reply:
non colorized icons are not yet included here.
the reasoning for those arguments is that I didn't want to hardcode a default color in the URL directly, I'd prefer having some leverage for later if I wanted to change that. (I'm not sure the red is the wanted default anyway...)

Reply:
OK, I see. So there'll be some logic added later on, that reads the color channels from the original icon file in case it belongs to the "non-colorizable" category or something similar.